docs/development/releasing/index.md

```diff
@@ -1,42 +1,53 @@
 # Releasing
 
-## Helm Charts
+As prerequisites to release:
+
+- The repository that holds the code for the component/package to release has:
+  - A release commit as last commit in `master` branch
+  - The last commit in master is tagged.
```

Comment:
Indentation is not being applied properly here. I'm not sure if the double indentations is intended.

<img width="677" alt="image" src="https://user-images.githubusercontent.com/8863238/94251611-0ae5ec00-ff23-11ea-952f-38d9aa362515.png">

Reply:
We are not always releasing from `master`, e.g. `0.6.1` was released from `maint-0.6`.  So you can use a neutral language such as "from the release branch such as `maint-0.7` for bug fixes or `master` for new versions)"

Reply:
You can also add to the list as the final step "The code base was tested locally in production conditions" without specifyng too much what that means for now...

Reply:
> We are not always releasing from master, e.g. 0.6.1 was released from maint-0.6. So you can use a neutral language such as "from the release branch such as maint-0.7 for bug fixes or master for new versions)"

This statement, while true, doesn't work with current `reana-dev` that's why the non-neutral language. I wrongly assumed we would document future behaviours as we would implement them (in https://github.com/reanahub/reana/issues/413 in this case). Amending...